jetson-xavier.coffee

```diff
@@ -0,0 +1,45 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+BOARD_PREPARE  = 'Put the NVidia Jetson Xavier board in recovery mode'
+FLASH_SCRIPT = 'Copy the flash_xavier.sh script near the BalenaOS image and execute: sudo ./flash_xavier.sh <path_to_balenaOS.img>'
+DONE_FLASHING  = 'Follow the instructions printed by the script'
+module.exports =
+	version: 1
+	slug: 'jetson-xavier'
+	aliases: [ 'jetson-xavier' ]
+	name: 'Nvidia Jetson Xavier'
+	arch: 'aarch64'
+	state: 'experimental'
+	community: true
```

Comment:
Was it discussed that we mark this as community? @shaunmulligan what are the implications for marking this as community since it's actually done by us instead?

Reply:
@floion Yes, that's what I recall talking with Shaun.